test/unittests/xmllib/models/internal/test_values.py

```diff
@@ -42,9 +43,9 @@ def test_warns(self) -> None:
             BooleanValue.new("other", ":booleanProp", permissions=Permissions.OPEN, resource_id="res_id", comment=None)
 
     def test_warns_potentially_empty_comment(self) -> None:
-        with pytest.warns(XmllibInputWarning):
+        with pytest.warns(XmllibInputInfo):
             BooleanValue.new(
-                "other", ":booleanProp", permissions=Permissions.OPEN, resource_id="res_id", comment=str(pd.NA)
+                True, ":booleanProp", permissions=Permissions.OPEN, resource_id="res_id", comment=str(pd.NA)
```

Comment:
Before, this raised 2 warnings:
1. XmllibInputWarning because "other" is not a valid boolean
2. XmllibInputInfo because `str(pd.NA)` is an empty comment

(1) is already tested above. The thing that this function wants to test is (2)